### PR TITLE
[561] Provide style guide references for each check

### PR DESCRIPTION
**Pull request by rnelson0**

### CODE REVIEW

File: lib/puppet-lint/plugins/check_whitespace.rb [hunk 1]
```diff
@@ -146,6 +158,7 @@ def check
         elsif token.type == :RBRACE || token.type == :SEMIC
           level_tokens[indent_depth_idx].each do |arrow_tok|
             unless arrow_tok.column == indent_depth[indent_depth_idx] || level_tokens[indent_depth_idx].size == 1
+puts indent_depth
```
Comment (rodjek) [inline lib/puppet-lint/plugins/check_whitespace.rb]: Looks like some debugging puts calls got committed?

File: lib/puppet-lint/plugins/check_whitespace.rb [hunk 2]
```diff
@@ -168,6 +181,7 @@ def check
 
   def fix(problem)
     new_ws_len = (problem[:indent_depth] - (problem[:newline_indent].length + problem[:token].prev_code_token.to_manifest.length + 1))
+puts "#{new_ws_len}, #{problem[:indent_depth]}, #{problem[:newline_indent].length}, #{problem[:token].prev_code_token.to_manifest.length + 1}, #{problem[:token].inspect}"
```
Comment (rodjek) [inline lib/puppet-lint/plugins/check_whitespace.rb]: Another debugging puts?

**Comment by rnelson0** [comment]
@rodjek Good catch, thanks! Not at all sure what mistake I made there, maybe time to rm/clone again, as one does :)

**Comment by ghoneycutt** [comment]
The style guide is also numbered, so you could say section 12.1 or whatever. I like the links though :)

**Pull request merged**